model/src/test/java/com/javahelp/model/user/ClientUserInfoTest.java

```diff
@@ -51,6 +53,32 @@ public void testSetAddress() {
         assertEquals("Bahen Center", c.getAddress());
     }
 
+    @Test(timeout = 50)
+    public void testSetSingleAttributeExist() {
+        c.setAttribute("Gender", "Male");
```

Comment:
With `Gender` pending PR, I think this would also need to change soon after that PR is approved, but it is still fine right now.

Reply:
Agreed